.circleci/config.yml

```diff
@@ -1235,120 +1273,120 @@ jobs:
-    jobs:
-      - orc8r-gateway-test
+  version: 2.1
+
+  # cloud:
+  #   jobs:
+  #     - insync-checkin
+  #     - cloud_lint
+  #     - cloud-test
+  #     - orc8r-build:
+  #         requires:
+  #           - insync-checkin
+  #           - cloud_lint
+  #           - cloud-test
+  #
+  # lib_gateway:
+  #   jobs:
+  #     - orc8r-gateway-test
 
   agw:
     jobs:
-      - lte-test
-      - session_manager_test
-      - mme_test
-      - mme-clang-tidy:
-          <<: *only_master
-      - mme-clang-warnings:
-          <<: *only_master
-      - lte-integ-test:
-          <<: *master_and_develop
-      - lte-agw-deploy:
-          <<: *only_master
-          requires:
-            - lte-test
-            - mme_test
-            - lte-integ-test
-      - c-cpp-codecov
-
-  feg:
-    jobs:
-      - feg-lint
-      - feg-precommit
-      - feg-build:
-          requires:
-            - feg-precommit
-
-  cwag:
-    jobs:
-      - cwag-precommit
-      - cwf-integ-test:
-          <<: *master_and_develop
-      - xwfm-test:
-          <<: *master_and_develop
-      - cwag-deploy:
-          <<: *only_master
-          requires:
-            - cwag-precommit
-            - cwf-integ-test
-      - xwfm-deploy:
-          <<: *only_master
-          name: xwfm-deploy-latest
-          requires:
-            - cwag-deploy
-            - xwfm-test
-          tag-latest: true
-      - cwag-deploy:
-          <<: *only_master
-          name: cwag-deploy-latest
-          requires:
-            - cwag-deploy
-            - xwfm-deploy-latest
-          images: 'gateway_python|gateway_pipelined|gateway_go'
-          tag-latest: true
-
-
-  cwf_operator:
-    jobs:
-      - cwf-operator-precommit
-      - cwf-operator-build:
-          requires:
-            - cwf-operator-precommit
-
-  hourly_xwfm:
-    triggers:
-      - schedule:
-          cron: "0 * * * *"
-          <<: *only_master
-    jobs:
-      - xwfm-test:
-          notify_magma_ci: false
-
-  nms:
-    jobs:
-      - nms-flow-test
-      - eslint
-      - nms-yarn-test:
-          requires:
-            - nms-flow-test
-      - nms-e2e-test:
-          requires:
-            - nms-flow-test
-            - eslint
-            - nms-yarn-test
-      - nms-build:
-          requires:
-            - nms-flow-test
-            - eslint
-            - nms-yarn-test
-            - nms-e2e-test
-
-  docusaurus:
-   jobs:
-    - docusaurus_build_and_deploy:
-        <<: *only_master
-
-  fossa:
-    jobs:
-      - fossa-analyze:
-          <<: *only_master
+      # - lte-test
+      # - session_manager_test
+      # - mme_test
+      # - mme-clang-tidy:
+      #     <<: *only_master
+      # - mme-clang-warnings:
+      #     <<: *only_master
+      # - lte-integ-test:
+      #     <<: *master_and_develop
+      # - lte-agw-deploy:
+      #     <<: *only_master
+      #     requires:
+      #       - lte-test
+      #       - mme_test
+      #       - lte-integ-test
+      - lte-agw-aws
```

Comment:
let's make it part of the `release` workflow

Reply:
you mean:
           <<: *only_master
?